### DISCUSSION TITLE
Laggy terminal startup on macOS because of repeated pyenv config

### DISCUSSION THREAD

**Discussion by realJogicodes** [pyenv/pyenv#3043] [General] [answered]
<img width="1470" alt="Screenshot 2024-09-02 at 10 41 44 AM" src="https://github.com/user-attachments/assets/3a41a969-699f-4b40-b5ab-713870c954fe">

I asked claude sonnet why my terminal startup is so slow every time supplying it with my .zshrc file. Saves me from digging manually. It identified these repetitive entries and tells me that once is enough and I agree with that statement. I have no clue how this happened, I noted that issue since pyenv installation so it must be somewhat related to the project.

418 lines of redundant configuration where it should be 2 only.

**Comment by realJogicodes** [comment, marked as answer]
Somehow, I ended up with an `echo` in my `.zshrc` that adds the configuration to the end of the `.zshrc`

I cannot say how that happened. Probably my own fuck up though, you can ignore this post or delete it. I was gonna but cannot find how to

**Reply by native-api** [reply]
Let's have this stay. You're not the first one who accidentally inserted the `echo <...> >>~/.bashrc` line into the rc files instead of executing it.
